style.css/secondsectionStyle.css

```diff
@@ -0,0 +1,16 @@
+.second-section {
+  text-align: center;
+}
+
+.title-second-section {
+  background-color: red;
+  color: #ffffff;
+  display: inline-block;
+  font-weight: bold;
+  padding: 8px;
+  margin-top: 40;
+}
+
+.text-second-section {
```

Comment:
Here also we need line-height. For all text, we'll need it. Since you add the reset CSS you need to put the correct line height based on your design.

Reply:
Done.